plugins/CorePluginsAdmin/templates/macros.twig

```diff
@@ -275,6 +258,19 @@
         {% endfor %}
         </tbody>
     </table>
+
+    <div class="footer-message">
```

Comment:
I recently got confused not to see plugin LanguagesManager in the plugins page, so listening now the ones that are always activated